tests/net/tcp2/prj.conf

```diff
@@ -50,3 +50,5 @@ CONFIG_NET_LOG=y
 #CONFIG_NET_IPV4_LOG_LEVEL_DBG=y
 #CONFIG_NET_IPV6_LOG_LEVEL_DBG=y
 #CONFIG_NET_CORE_LOG_LEVEL_DBG=y
+
+CONFIG_NET_TCP_RECV_QUEUE_TIMEOUT=1000
```

Comment:
It would be better to test both zero and non-zero configuration, by extending `testcase.yaml` to test both. Of course introduced "out of order packets" test scenarios should be skipped in case of `CONFIG_NET_TCP_RECV_QUEUE_TIMEOUT=0`.

Reply:
We can add two test scenarios to yaml file although it seems a bit pointless as in this case we just do not run the two new test cases.